app/views/dashboard/_hosts_in_build_mode_widget.html.erb

```diff
@@ -0,0 +1,11 @@
+<% hosts = @data.hosts.in_build_mode.includes(:token).preload(:owner).order(created_at: :desc).limit(9) %>
```

Comment:
This query must include both "in build" and also when "build errors" text is present (not empty).